packit_service/worker/events/github.py

```diff
@@ -204,6 +222,14 @@ def get_dict(self, default_dict: Optional[Dict] = None) -> dict:
         result["issue_id"] = self.issue_id
         return result
 
+    @property
+    def comment_object(self) -> Optional[IssueComment]:
+        if not self._comment_object:
+            self._comment_object = self.project.get_issue(self.issue_id).get_comment(
```

Comment:
since at the end of processing of the event we send the event as a JSON to handler Celery task, the dictionary from `Event.get_dict()` needs to be JSON serializable (I think it is not really tested in other events, so I will probably create an issue for it), so there can be a problem with this attribute and I think we need to call `event_dict.pop("_comment_object")` in `get_dict()` method for these classes (please, check how it is done in some other Event subclasses (e.g.we are removing the `project` as well)